src/libraries/Silk.Net.Windowing/Implementations/GLFW/GLFWSurface.cs

```diff
@@ -0,0 +1,88 @@
+﻿using Silk.NET.Core;
+using Silk.NET.Maths;
+
+namespace Silk.NET.Windowing.Implementations.GLFW;
+
+public class GLFWSurface : IGLTransparentFramebuffer, IGlesDesktopSurface, IVkDesktopSurface, IGLDesktopSurface
```

Comment:
This doesn't seem like it'll work 🤔 we will see when implementing I guess

Reply:
Why would it not work?